### PR TITLE
Adding media documentation

### PR DESCRIPTION
**Pull request by vijaynandwani**
The documentation can be viewed here: https://isthegeek.gitbooks.io/d8-guide/content/modules/media/intro.html

I have thought of adding two more sections:
1) Configuring the embed button
2) Contributing to the project

Please let me know if there are any more sections which should be added. Also if there is some other page which should be updated.

Thanks

### CODE REVIEW

**Comment by vijaynandwani** [comment]
I think I have added all the sections now. Let me know if you want me add/edit anything. Thanks

**Comment by mariacha** [comment]
This documentation was very helpful for me -- just adding my two cents. :)

File: SUMMARY.md [hunk 1]
```diff
@@ -5,49 +5,54 @@
...
-       * [Instagram](modules/media_entity/instagram.md)
-   * [Crop API](modules/crop/intro.md)
-       * [Architecture](modules/crop/architecture.md)
-       * [Implementation examples](modules/crop/Implementation_examples.md)
-   * [ImageWidgetCrop](modules/image_widget_crop/intro.md)
-   * [Fallback Formatter]((modules/fallback_formatter/intro.md))
-   * [Field formatter](modules/field_formatter/intro.md)
-   * [Flysystem](modules/flysystem/intro.md)
-   * [Image Effects](modules/image_effects/intro.md)
+    * [Entity Browser](modules/entity_browser/intro.md)
```
Comment (davereid) [inline SUMMARY.md:14]: These diffs seem unrelated, could we leave in the existing spacing and fix it in a follow-up PR?

**Comment by slashrsm** [comment]
I fixed few obvious things and merged it. Since GSoC is over it is unlikely that someone would spend significant time on this pull. It is still better than nothing. Let's add more fixes in follow-ups.

**Pull request merged**